src/lib.rs

```diff
@@ -28,3 +28,13 @@ pub mod poll;
 pub mod prelude;
 pub mod story;
 pub mod user;
+
+pub use ask::Ask;
+pub use client::Client;
+pub use comment::Comment;
+pub use error::HError;
+pub use item::Item;
```

Comment:
This would require additional changes (I believe?) since `Item` is `pub(crate)`. I've been using it as an internal representation for now, but if your use case requires it to be public, I am open to changing it.

Reply:
Yeah, you're right. `Item` is not a public interface. I'll change it...